MonoDevelop.MSBuild.Editor/Completion/MSBuildCompletionSource.cs

```diff
@@ -482,6 +599,10 @@ enum MSBuildSpecialCommitKind
 		NewGuid,
 		PropertyReference,
 		ItemReference,
+		Element,
+		Attribute,
+		AttributeValue,
+		AttributeValueSpecial, //for package reference attribute values specifically
```

Comment:
`Special` is not very meaningful, I would suggest something like `PackageReferenceAttributeValue`

Reply:
I changed it in the recent commit. Pushing more changes soon.